mysql-test/t/bug_ps7923-master.opt

```diff
@@ -0,0 +1,5 @@
+--early-plugin-load="keyring_file=$KEYRING_PLUGIN"
+--loose-keyring_file_data=$MYSQL_TMP_DIR/keyring
+$KEYRING_PLUGIN_OPT
+--gtid-mode=ON --enforce-gtid-consistency=ON
+--innodb-use_native_aio=OFF
```

Comment:
Is this `--innodb-use_native_aio=OFF` really needed?

Reply:
No, I don't think it was ever needed for the test. Removed the parameter.